pkg/packagekit/updates.jsx

```diff
@@ -592,6 +592,7 @@ const ApplyUpdates = ({ transactionProps, actions, onCancel, rebootAfter, setReb
                         &nbsp;{curPackage}
                     </div>
                     <Progress title={remain}
+                              aria-label={remain ? _("Time remaining: ") : _("Update progress")}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22377-77217c12-20250829-142815-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)